static/src/javascripts/projects/common/modules/identity/sign-in-gate/variants/dismiss-window/variant-2-day.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+import type { CurrentABTest, SignInGateVariant } from '../../types';
+import { componentName } from '../../component';
+import {
+    hasUserDismissedGateInWindow,
+    isNPageOrHigherPageView,
+    isLoggedIn,
+    isInvalidArticleType,
+    isInvalidSection,
+    isIOS9,
+    setGatePageTargeting,
+} from '../../helper';
+
+// pull in the show method from the design folder, which has the html template and and click handlers etc.
+import { designShow } from '../design/main-variant';
+
+// define the variant name here
+const variant = 'dissmiss-window-variant-2-day';
```

Comment:
I'm slightly worried that the variant name would be a bit confusing, since `2-day` makes it sounds like its a "two day" window, rather than being the 2nd variant. 🤔

Reply:
hmmm... possibly, but how long is this test going to be around to confuse people other than us? I can just add more inline comments